changelogs/unreleased/gh-6305-net-box-autocomplete.md

```diff
@@ -0,0 +1,4 @@
+## feature/net.box
```

Comment:
There's no such section as 'net.box':

https://github.com/tarantool/tarantool/blob/b99ac0b4abd2b1beaf7e14123107fd07fb8db4ec/tools/gen-release-notes#L14-L17

Let's use `feature/lua` instead.

Reply:
done. 
let's add some checker in checkpatch?

Reply:
Actually, the [release-notes check](https://github.com/tarantool/tarantool/runs/5587899797?check_suite_focus=true) shows the Netbox clause in release notes

#### Netbox

* Added `return_raw` net.box option for returning `msgpack.object` instead of
  decoding the response (gh-4861).

Reply:
Too bad. There's also Net.box section...

```
#### Netbox

* Added `return_raw` net.box option for returning `msgpack.object` instead of
  decoding the response (gh-4861).

...

### Net.box

* Changed the type of the error returned by net.box on timeout
  from ClientError to TimedOut (gh-6144).
```

We need to fix it and add a check to tools/gen-release-notes.